stream/src/public_key/handshake.rs

```diff
@@ -4,16 +4,10 @@ use bytes::Bytes;
 use commonware_cryptography::{PublicKey, Scheme};
 use commonware_macros::select;
 use commonware_runtime::{Clock, Sink, Spawner, Stream};
-use commonware_utils::{union, SystemTimeExt};
+use commonware_utils::SystemTimeExt;
 use prost::Message;
 use std::time::{Duration, SystemTime};
 
-const NAMESPACE_SUFFIX_HANDSHAKE: &[u8] = b"_HANDSHAKE";
```

Comment:
Given we only sign one type of message, this ends up being more confusing than its worth IMO. We can rethink if we add new messages (which seems unlikely).